app/process/places/places.lua

```diff
@@ -36,19 +34,18 @@ local function ExitProcessing(object)
 end
 
 local function processTags(tags)
-  local allowed_tags = Set({ "name", "place", "capital", "website", "wikidata", "wikipedia", "population",
+  local tags_cc = Set({ "name", "place", "capital", "website", "wikidata", "wikipedia", "population",
     "population:date", "admin_level" })
-  FilterTags(tags, allowed_tags)
+  -- TODO: I don't think that we need the line bellow, osm2pgsql should convert this automatically
```

Comment:
Du meinst das tonumber? In population kann halt auch anderer Kram stehen wie "ca 200" den wir damit löschen.

Reply:
Ja, ist mir dann auch aufgefallen